src/components/BrandConfigProvider/icons/IconProvider.tsx

```diff
@@ -1,16 +1,20 @@
 import React, { createContext, useContext } from 'react';
 import * as Chromicons from '@lifeomic/chromicons-native';
+import { TriangleFilled } from './TriangleFilled';
 
-export type Icons = typeof Chromicons & Record<string, React.ComponentType>;
+export type Icons = typeof Chromicons &
+  Record<string, React.ComponentType> & {
+    TriangleFilled: typeof TriangleFilled;
+  };
 
-const IconContext = createContext<Icons>(Chromicons);
+const IconContext = createContext<Icons>({ ...Chromicons, TriangleFilled });
```

Comment:
This seems totally fine for this PR but it would be nice if we could just add it to Chromicons in a future version. I see a lot of these one-off icons and it feels like we would get a lot more benefit making them available via Chromicons.

Reply:
Yeah, great point. I will follow up with a PR into Chromicons.